packages/wallet/src/redux/channelState/challenging/__tests__/challenging.test.ts

```diff
@@ -70,7 +70,7 @@ describe('when in INITIATE_CHALLENGE', () => {
   const state = states.waitForChallengeInitiation(defaults);
 
   describe('when a challenge is initiated', () => {
-    const action = actions.transactionSentToMetamask(channelId);
+    const action = actions.transactionSentToMetamask(channelId, 'DirectFunding');
```

Comment:
I'd prefer this to be a named constant.

Reply:
This is obsolete now but it was typed to a `WalletProcess` type so it would only accept 'DirectFunding` or `IndirectFunding`

Reply:
I agree, but if you want to change `DirectFunding` to something else, currently you have to replace-in-all-files and hope you don't hit a false positive.

If it is a constant that is exported, you rename the constant and you're good.

Reply:
I've changed `WalletProcedure` to a string enum.